src/tls.c

```diff
@@ -4697,6 +4700,33 @@ int TLSX_ValidateSupportedCurves(WOLFSSL* ssl, byte first, byte second) {
             }
         }
 
+        /* REVIEWER: ECDHE_PSK_WITH_AES_128_CBC_SHA256 is not present in this
```

Comment:
I don't understand this comment. Are you saying this `TLSX_ValidateSupportedCurves` isn't used for the `TLS_ECDHE_PSK_WITH_AES_128_GCM_SHA256` case? Spell: `necessary`.

Reply:
This comment will be removed.  I'm saying that the case of `ECDHE_PSK_WITH_AES_128_CBC_SHA256` is no handled in `TLSX_ValidateSupportedCurves()` . I'm wondering if it should or needs to be handled?